front/public/locales/fr/simulation.json

```diff
@@ -67,6 +67,7 @@
   "stopStopTime": "Arrêt",
   "stopTime": "Arrivée",
   "timetable": "Table horaire",
+  "timetableOutput": "Table horaire de sortie",
```

Comment:
In the mockup there is:
"timetableOutput": "Table de sortie",

Reply:
I know as I said above, but there was already a translation key to "table horaire", the title of the corresponding input section is "horaires et arrets", and I don't know if the title in the mockup was set taking the page into account and not just the component, which the title is not really a part of. Same reason why I put the same style as the title already set above.

There is also a button next to it in the mockup, but it's not implemented, and so I don't think it makes sense to try and replicate this part of the mockup now. I think setting properly all the elements and buttons outside the table and uniformizing the sections in the page should probably be another mockup and pass. This is also why I didn't add any title in the first place.